automatic/git.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,25 @@
+﻿$ErrorActionPreference = 'Stop'
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$filePath32  = "$toolsPath\Git-2.11.0-32-bit.exe"
+$filePath64  = "$toolsPath\Git-2.11.0-64-bit.exe"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$pp = Get-PackageParameters
+Set-InstallerRegistrySettings $pp
+
+Stop-GitSSHAgent
+
+$packageArgs = @{
+    PackageName    = 'git.install'
+    FileType       = 'exe'
+    SoftwareName   = 'Git version *'
+    File           = $installFile
+    SilentArgs     = "/VERYSILENT", "/SUPPRESSMSGBOXES", "/NORESTART", "/NOCANCEL", "/SP-", "/LOG", (Get-InstallComponents $pp)
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
```

Comment:
> And in that case you get something on PATH even you don't want it. This is so confusing that even I don't know what param to use here.

To install git with no params and expect `git` to work immediately is always desired. I did remove the install PATH at one point and so many people complained I put it back. Many packages do this for most installers. If they do not add PATH, lots of packages will add it - see VirtualBox and other packages. Plus